addons/web/static/tests/views/form_tests.js

```diff
@@ -2044,10 +2044,11 @@ QUnit.module('Views', {
     QUnit.test('make default record with non empty one2many', async function (assert) {
         assert.expect(4);
 
+        this.data.partner.records = [];
```

Comment:
?

Reply:
Good challenge, after a pretty long investigation, it turns out when command 6 is given as default, the mock server was simply taking the list from the command as value, which means all records would share the same.